docs/windows.md

```diff
@@ -421,59 +316,173 @@ Nodes and Pods in your cluster by running:
 ```
 
-### Add Windows kube-proxy DaemonSet (only for Kubernetes versions prior to 1.26)
+### Installation as a Service
+
+Install Antrea (v0.13.0+ is required for containerd) as usual. The following
+command deploys Antrea with the version specified by `<TAG>`:
+
+```bash
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
+```
+
+When running the Antrea Agent as a Windows service, no DaemonSet is created for
+Windows worker Nodes. You will need to ensure that [nssm](https://nssm.cc/) is
+installed on all your Windows Nodes. `nssm` is a handy tool to manage services
+on Windows.
+
+To prepare your Windows worker Nodes, follow the steps in [Join Windows worker Nodes](#join-windows-worker-nodes).
+With this installation method, OVS daemons are always run as services (not
+containerized), and you will need to run `Install-OVS.ps1` to install them.
+
+When your Nodes are ready, run the following scripts to install the antrea-agent
+service. NOTE: `<KubernetesVersion>`, `<KubeconfigPath>` and
+`<KubeletKubeconfigPath>` should be set by you. `<KubeProxyKubeconfigPath>` is
+an optional parameter that is specific to kube-proxy mode. For example:
+
+```powershell
+# kube-proxy mode is no longer supported starting with K8s version 1.26
+$InstallKubeProxy=$false
+$KubernetesVersion="v1.23.5"
+$KubeConfig="C:/Users/Administrator/.kube/config" # admin kubeconfig
+$KubeletKubeconfigPath="C:/etc/kubernetes/kubelet.conf"
+if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="C:/Users/Administrator/kubeproxy.conf" }
+```
 
-Starting from Kubernetes 1.26, Antrea will no longer support Windows kube-proxy
-because kube-proxy userspace mode is deprecated and kernelspace mode does not
-work with Antrea. For a cluster with Kubernetes version 1.26 or later, please
-refer to [Deploying Antrea on Windows Worker Node](#deploying-antrea-on-windows-worker-node)
+```powershell
+$TAG="v1.14.0"
+$KubernetesVersion="<KubernetesVersion>"
+$KubeConfig="<KubeconfigPath>"
+$KubeletKubeconfigPath="<KubeletKubeconfigPath>"
+if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="<KubeProxyKubeconfigPath>" }
+$KubernetesHome="c:/k"
+$AntreaHome="c:/k/antrea"
+$KubeProxyLogPath="c:/var/log/kube-proxy"
 
-First, deploy a Windows-compatible version of kube-proxy. You can download `kube-proxy.yaml`
-from the Kubernetes github repository to deploy kube-proxy. The kube-proxy version
-in the YAML file must be set to a Windows compatible version. The following command
-downloads and applies `kube-proxy.yml`:
+curl.exe -LO "https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Helper.psm1"
+Import-Module ./Helper.psm1
+
+Install-AntreaAgent -KubernetesVersion "$KubernetesVersion" -KubernetesHome "$KubernetesHome" -KubeConfig "$KubeConfig" -AntreaVersion "$TAG" -AntreaHome "$AntreaHome"
+New-KubeProxyServiceInterface
+
+New-DirectoryIfNotExist "${AntreaHome}/logs"
+New-DirectoryIfNotExist "${KubeProxyLogPath}"
```

Comment:
line 376 and line 379 are supposed to be under the block `if ($InstallKubeProxy)` in line 381 ?

Reply:
I copied this script as it was before. I didn't make any change even though that may not be obvious from the Github diff.
I think you're right but on the other hand, it should not be an issue to keep things as they are (even if we create the interface for nothing)?

Reply:
No issues were introduced to create the new interface and directories, just not necessary. Maybe we can remove them when kube-proxy is removed from the documentation.